docassemble/PDFStats/pdfstats.py

```diff
@@ -119,50 +151,55 @@ def view_stats(name):
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Statistics for { name }</title>
+    <title>Stats for { title }</title>
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous">
     <style>
     .suffolk-blue {{
         background-color: #002e60;
     }}
     </style>
   </head>
-<body class="text-center">
+<body>
 <nav class="navbar navbar-dark suffolk-blue">
   <div class="container-fluid">
     <a class="navbar-brand" href="https://suffolklitlab.org">
   <img src="https://apps.suffolklitlab.org/packagestatic/docassemble.MassAccess/lit_logo_light.png?v=0.3.0" alt="Logo" width="30" height="24" class="d-inline-block align-text-top"/>
   Suffolk LIT Lab
   </a>
   </div>
-</nav>    
+</nav>
 
-<main style="max-width: 500px; margin: auto;">
-<h1 class="pb-2 border-bottom">File statistics for <span class="text-break">{ name }</span></h1>
-<table class="table">
+<main style="max-width: 800px; margin-left: auto; margin-right: auto;">
+<h1 class="pb-2 border-bottom text-center">File statistics for <span class="text-break">{ title }</span></h1>
+<p><a href="/pdfstats/download/{file_id}">Download the file</a></p>
+<br/>
+<table class="table text-center">
     <thead>
     <tr>
       <th scope="col">Statistic name</th>
       <th scope="col">Value</th>
-      <th scope="col">Benchmark</th>
+      <th scope="col">Note</th>
```

Comment:
That's fair, I had been adding more clarifying notes in that column instead of just the single number score, I get that note is vague though. maybe "why?"

Reply:
Changed this to `Target + Compare` in the flask branch.